### PR TITLE
Build multiarch packages, include license files, and some other minor fixes

### LINKED ISSUE #12: @conda-forge-admin, please re-render
@conda-forge-admin, please re-render.

### PR DESCRIPTION
**Pull request by conda-forge-linter**
Hi! This is the friendly automated conda-forge-webservice.

I've rerendered the recipe as instructed in #12.

Here's a checklist to do before merging.
- [x] Bump the build number if needed.

Fixes #12
Fixes https://github.com/conda-forge/binutils-feedstock/issues/14

### CODE REVIEW

**Comment by jakirkham** [comment]
@jayfurmanek, appears the re-rendering was missing from PR ( https://github.com/conda-forge/binutils-feedstock/pull/11 ). So this now adds aarch64 and ppc64le support.

**Comment by jayfurmanek** [comment]
ah ok. Needs `make`.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,6 +21,8 @@ requirements:
     - {{ compiler('cxx') }}
     - bison
     - flex
+    - make
+  host:
```
Comment (jayfurmanek) [inline recipe/meta.yaml]: empty `host` stanza here is strange. Needed?

**Comment by jakirkham** [comment]
@conda-forge/core, do we know what's up with the linter? It was [more-or-less ok]( https://github.com/conda-forge/binutils-feedstock/pull/13#issuecomment-532884260 ), but [now it seems to barf]( https://github.com/conda-forge/binutils-feedstock/pull/13#issuecomment-533200724 ).

**Comment by isuruf** [comment]
Please see the comment https://github.com/conda-forge/binutils-feedstock/pull/13#issuecomment-533200724. Have you run `conda smithy recipe-lint .` locally?

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml]: IIRC it was needed to ensure that `build` and `host` were two different environments. This ensures installing the library doesn't clobber existing content from other packages used in the build, which would result in some content missing in the final package. That said, maybe conda-build has changed and this is no longer needed any more? Sorry it's hard to keep track.

**Comment by jakirkham** [comment]
It seems like the linter is getting overly pedantic here. This did not use to cause an issue with the linter and conda-build is not bothered by it.

```python
$ conda smithy recipe-lint recipe/
Traceback (most recent call last):
  File "/Users/jkirkham/miniconda/bin/conda-smithy", line 10, in <module>
    sys.exit(main())
  File "/Users/jkirkham/miniconda/lib/python3.7/site-packages/conda_smithy/cli.py", line 477, in main
    args.subcommand_func(args)
  File "/Users/jkirkham/miniconda/lib/python3.7/site-packages/conda_smithy/cli.py", line 372, in __call__
    return_hints=True,
  File "/Users/jkirkham/miniconda/lib/python3.7/site-packages/conda_smithy/lint_recipe.py", line 574, in main
    results, hints = lintify(meta, recipe_dir, conda_forge)
  File "/Users/jkirkham/miniconda/lib/python3.7/site-packages/conda_smithy/lint_recipe.py", line 392, in lintify
    for requirement in requirements:
TypeError: 'NoneType' object is not iterable
```

**Comment by isuruf** [comment]
> This did not use to cause an issue with the linter.

What do you mean by this?

**Comment by jakirkham** [comment]
The empty `host` section has not caused a problem for the linter previously. Now it is causing it an issue as the exception implies. Unless of course I'm misreading things here?

**Comment by isuruf** [comment]
Ah, yes. Broken by https://github.com/conda-forge/conda-smithy/pull/1133. Can you open an issue or a PR?

**Comment by jakirkham** [comment]
Thanks for spotting that @isuruf. Sorry for being unclear initially.

Sure have raised issue ( https://github.com/conda-forge/conda-smithy/issues/1153 ) to track this.

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml]: Yeah let's go ahead and back this out. Was waiting to see if the build would have other errors we needed to fix before changing, but [it looks ok]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=73706 ).

**Comment by jakirkham** [comment]
This is now passing! @frol, could you please take a look? 🙂

**Comment by frol** [comment]
@jakirkham Thank you!

**Pull request merged**